app/oauth/rest.py

```diff
@@ -109,6 +111,15 @@ def authorize():
                 return make_response(redirect(f"{current_app.config['UI_HOST_NAME']}/login/failure"))
 
 
+@oauth_blueprint.route('/my-services/<uuid:user_id>', methods=['GET'])
```

Comment:
I've validated that this works, but was wondering if it really needed to be in the `auth` endpoint or if this should be in the `users` endpoint since it is getting a users data?